### PR TITLE
Updated code review checklist

### PR DESCRIPTION
**Pull request by feerrenrut**
### Link to issue number:
None

### Summary of the issue:
After internal discussion, some additional items and clarifications have been identified to add to the the code review checklist.

### Description of how this pull request fixes the issue:
Adds a new item for authors and reviewers to consider different UX modalaites.
- It is common to focus on Speech output and forget to consider Braille, and Low vision UX.
  Additionally, a reminder about supporting multiple browsers.
- Clarifies the manual test item, reviewers should be (where possible) verifying the behaviour of the PR.
  This helps to clarify that the description of the PR is complete, and that the reviewer understands how the change affects users.

While here I have tried to clarify the wording for the short code review explanation.
Not all items on the checklist will apply to every change, this list is purely a reminder to authors and reviewers that such items are commonly fogotten.
The template just asks that the author and reviewer confirm that they have thought about each item and whether it is applicable to the PR.

Additionally I have copied the [coding standards from our wiki](https://github.com/nvaccess/nvda/wiki/Contributing#code-style) to the devDocs folder.
By storing this in the repository, it is more likely to be found, read, and kept up to date.
Once merged, we can link to it from the wiki page.
Now, when there is an idiomatic solution to a mistake in code we can request it is added to the codingStandards document as part of the PR.

The Issue template and PR template explainer docs have also been moved from the wiki. This should make it easier for us to keep these in sync, and now we can accept contributions to clarify.

### Testing strategy:
Checked the rendered markdown output

### Change log entries:
New features
Changes
Bug fixes
For Developers

### Code Review Checklist:

- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual testing / Reviewer verification.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different browsers

### CODE REVIEW

File: devDocs/codingStandards.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* Boolean functions or variables
+  - Prefer positive form of the language.
+    Avoid double negatives like `shouldNotDoSomething = False`
+  - Start with a "question word" to hint at their boolean nature. EG `shouldX`, `isX`, `hasX`
+* Classes should use mixed case to separate words, starting with an upper case letter;
+  - E.G. `BrailleHandler`.
+* Constants should be all upper case, separating words with underscores;
+  - E.G. `LANGS_WITH_CONJUNCT_CHARS`.
+* Event handlers are prefixed with "event_", subsequent words in capitals using snake case.
+  - E.G.: `event_ACTION` or `event_OBJECT_ACTION`.
```
Comment (leonardder) [inline devDocs/codingStandards.md:48]: Is this true? event_gainFocus uses event_lowerCamelCase

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -25,16 +25,24 @@ For Developers
...
-Authors, please do a self-review and confirm you have considered the following items.
-Mark items you have considered by checking them.
-You can do this when editing the Pull request description with an x: `[ ]` becomes `[x]`.
+Authors, please do a self-review of this pull-request.
+Check items to confirm you have thought about the relevance of the item.
+Where items are missing (eg unit / system tests), please explain in the PR.
+To check an item `- [ ]` becomes `- [x]`, note spacing.
 You can also check the checkboxes after the PR is created.
+A detailed explanation of this checklist is available here:
+https://github.com/nvaccess/nvda/wiki/Github-pull-request-template-explanation-and-examples#code-review-checklist
```
Comment (seanbudd) [inline .github/PULL_REQUEST_TEMPLATE.md:34]: We must make sure to update this link with more information on "UX of all users considered" and what "Reviewer verification" means.

**Review by seanbudd — APPROVED** [review]
LGTM

File: devDocs/codingStandards.md (hunk 1 above)
Comment (feerrenrut) [reply devDocs/codingStandards.md:48]: Hmm good point I'll update that.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -25,16 +25,24 @@ For Developers
...
 -->
 
 - [ ] Pull Request description is up to date.
 - [ ] Unit tests.
 - [ ] System (end to end) tests.
-- [ ] Manual tests.
+- [ ] Manual testing / Reviewer verification.
```
Comment (michaelDCurran) [inline .github/PULL_REQUEST_TEMPLATE.md:40]:
```suggestion
- [ ] Manual testing (including by reviewer where possible) 
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -25,16 +25,24 @@ For Developers
...
 - [ ] User Documentation.
 - [ ] Change log entry.
 - [ ] Context sensitive help for GUI changes.
+- [ ] UX of all users considered:
+  - Speech 
+  - Braille
+  - Low Vision
+  - Different browsers
```
Comment (michaelDCurran) [inline .github/PULL_REQUEST_TEMPLATE.md:48]:
```suggestion
  - Different web browsers
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (feerrenrut) [reply .github/PULL_REQUEST_TEMPLATE.md:40]: I think I'd prefer to keep the item short and explain it on the wiki. All items are "where possible".

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (feerrenrut) [reply .github/PULL_REQUEST_TEMPLATE.md:34]: These docs could actually be moved to the developer docs in the repository. Then changes could be paired? @michaelDCurran what do you think?

**Comment by michaelDCurran** [comment]
I'm fine with them being moved to devDocs, yes.

**Comment by michaelDCurran** [comment]
Short is fine, but I really don't like the term "reviewer verification" 
especially if it is after a / (slash) which suggests something paralell, 
not extra information.

I actually think, if we are the ones reviewing, then we already know 
this, and I think simply saying "Manual testing" is enough, as we know 
it applies to us as well.

**Review by seanbudd — APPROVED** [review]

**Pull request merged**